app/controllers/AssessmentScoresController.scala

```diff
@@ -117,3 +127,26 @@ trait AssessmentScoresController extends BaseController {
     }
   }
 }
+
+object AssessorAssessmentScoresController extends AssessmentScoresController {
+  val service: AssessmentScoresService = AssessorAssessmentScoresService
+  val auditService: AuditService = AuditService
+  val repository: AssessmentScoresRepository = repositories.assessorAssessmentScoresRepository
+  val AssessmentScoresAllExercisesSaved = "AssessorAssessmentScoresAllExercisesSaved"
+  val AssessmentScoresOneExerciseSaved = "AssessorAssessmentScoresOneExerciseSaved"
+  val UserIdForAudit = "reviewerId"
+
+  override def submitFinalFeedback() = Action.async(parse.json) {
```

Comment:
Interesting choice on placing this method in the trait and overriding it here. I don't see the need for this here - it creates and then breaks a contract. Can this method not be placed in the ReviewerAsessmentScoresController directly or (perhaps better) another trait and mixed-in?